File: 프로그래머스/unrated/181892. n 번째 원소부터/n 번째 원소부터.cpp
#include <string>
#include <vector>

using namespace std;
vector<int> ret;
vector<int> solution(vector<int> num_list, int n) {
    for(int i = n - 1; i < num_list.size(); i++) ret.push_back(num_list[i]);
    return ret;
}

File: 프로그래머스/0/181892. n 번째 원소부터/n 번째 원소부터.java
import java.util.*;
import java.util.stream.*;
class Solution {
    public int[] solution(int[] num_list, int n) {
        List<Integer> l = new ArrayList<>();
        for(int i = n-1; i < num_list.length; i++) l.add(num_list[i]);
        return l.stream().mapToInt(Integer::intValue).toArray();
    }
}